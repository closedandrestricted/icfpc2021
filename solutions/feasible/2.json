{"vertices":[[16,16],[13,12],[7,18],[21,23],[14,22],[6,15],[10,14],[12,14],[27,30],[10,7],[7,11],[16,14],[12,7],[5,15],[7,15],[23,16],[7,14],[9,19],[9,7],[30,13],[9,9],[13,10],[5,21],[33,26],[10,11],[12,10],[14,5],[16,26],[14,9],[8,11],[6,12],[11,7],[14,16],[5,12],[13,11],[5,6],[11,14],[17,16],[29,16]]}