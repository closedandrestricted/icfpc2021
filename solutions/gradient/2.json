{"vertices":[[18,56],[15,60],[22,47],[27,32],[26,54],[12,23],[13,7],[11,6],[35,36],[17,16],[15,11],[35,50],[16,12],[27,5],[29,6],[43,48],[24,16],[25,11],[38,42],[50,45],[35,9],[38,38],[30,21],[60,54],[48,7],[46,7],[51,12],[61,37],[52,6],[65,7],[66,5],[54,10],[52,10],[65,10],[58,12],[72,6],[62,12],[64,25],[75,25]]}