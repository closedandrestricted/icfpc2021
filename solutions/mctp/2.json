{"vertices":[[13,7],[8,7],[17,16],[32,16],[7,17],[49,15],[51,5],[49,5],[27,9],[54,10],[55,5],[17,18],[54,10],[72,15],[72,17],[22,24],[61,13],[66,15],[9,24],[29,25],[70,11],[12,21],[60,19],[22,36],[49,37],[50,35],[44,36],[23,53],[51,34],[42,41],[41,43],[48,36],[46,40],[44,39],[52,39],[46,39],[56,37],[34,58],[40,49]]}