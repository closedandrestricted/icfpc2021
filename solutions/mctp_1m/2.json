{"vertices":[[56,35],[60,32],[46,32],[36,40],[52,39],[29,25],[21,14],[22,16],[31,49],[24,19],[19,17],[43,40],[23,10],[20,19],[22,20],[37,44],[17,16],[14,20],[52,40],[40,38],[20,25],[48,40],[24,15],[52,37],[47,15],[46,17],[45,10],[64,26],[48,11],[54,6],[52,7],[47,7],[41,13],[53,9],[47,12],[46,6],[49,15],[71,19],[68,10]]}