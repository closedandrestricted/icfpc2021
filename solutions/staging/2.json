{"vertices":[[31,30],[27,27],[27,22],[42,17],[17,24],[35,34],[23,44],[23,46],[51,12],[31,42],[27,45],[22,16],[19,52],[27,49],[27,47],[28,10],[26,47],[21,47],[14,10],[35,10],[23,51],[17,13],[31,41],[46,10],[61,56],[61,54],[54,45],[59,19],[59,52],[49,41],[47,41],[57,48],[58,48],[49,44],[57,43],[47,47],[55,39],[67,11],[66,23]]}