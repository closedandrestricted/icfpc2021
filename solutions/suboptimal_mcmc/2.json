{"vertices":[[67,10],[72,7],[57,10],[44,10],[65,15],[26,10],[13,7],[14,5],[35,10],[19,13],[15,10],[62,23],[7,6],[6,23],[8,23],[54,21],[12,13],[8,17],[70,25],[47,23],[14,25],[65,25],[19,16],[48,36],[30,43],[31,45],[31,36],[54,50],[27,40],[24,51],[25,49],[31,40],[36,43],[27,51],[31,45],[23,55],[33,49],[63,58],[52,55]]}