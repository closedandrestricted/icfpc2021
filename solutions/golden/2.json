{"vertices":[[17,59],[22,56],[24,52],[23,37],[34,54],[13,22],[12,6],[10,5],[31,42],[16,14],[13,10],[44,55],[15,10],[33,8],[32,6],[50,51],[23,14],[27,10],[54,58],[45,45],[26,6],[59,58],[29,18],[57,48],[48,7],[46,8],[51,12],[59,34],[52,6],[66,8],[66,6],[54,10],[52,11],[65,11],[58,13],[72,6],[62,12],[65,25],[75,25]]}